IPython/html/static/edit/js/main.js

```diff
@@ -31,11 +31,14 @@ require([
 
     var base_url = utils.get_body_data('baseUrl');
     var file_path = utils.get_body_data('filePath');
-    contents = new contents.Contents({base_url: base_url});
     var config = new configmod.ConfigSection('edit', {base_url: base_url});
     config.load();
     var common_config = new configmod.ConfigSection('common', {base_url: base_url});
     common_config.load();
+    contents = new contents.Contents({base_url:base_url, 
+                                        config:config,
```

Comment:
Is there any reason to pass `config` as well? Our own contents class doesn't use any config, and in general I don't want to be passing config to an API module except where it's really necessary.